### PR TITLE
extracting the css to a separate file so it works for server side rendering

### PR DESCRIPTION
**Pull request by alejandromaka**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -17,11 +18,14 @@ module.exports = {
     module: {
         loaders: [
             {test: /\.jsx$/, loader: 'babel-loader', query: {plugins: ['transform-object-assign']}},
-            {test: /\.css$/, loader: 'style!css?localIdentName=[name]__[local]___[hash:base64:5]!postcss'}
+            {test: /\.css$/, loader: ExtractTextPlugin.extract('style-loader', 'css?localIdentName=[name]__[local]___[hash:base64:5]!postcss')}
```
Comment (hellatan) [inline webpack.config.js]: style-loader is needed here? I guess i should have asked the same question in the other PR.

**Comment by hellatan** [comment]
you should add `dist` to the `.gitignore` file and not check that stuff in. You should also create a new `.npmignore` file. That file can be empty.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Usage:
 ```
 const Tooltip = require('dibs-react-tooltip');
-
+require('dibs-react-tooltip/dist/main.css'); // to include default styles
```
Comment (hellatan) [inline README.md]: shouldn't `require('dibs-react-tooltip')` be enough for this? shouldn't that be bundled from there?

Comment (kostasmanionis) [inline README.md]: `dibs-react-tooltip/dist/main.css` isn't required in this package, so it wouldn't get bundled

File: webpack.config.js (hunk 1 above)
Comment (kostasmanionis) [inline webpack.config.js]: Yeah, we probably don't need it

Comment (alejandromaka) [inline webpack.config.js]: so empty string?

Comment (alejandromaka) [inline webpack.config.js]: crap didnt work

Comment (alejandromaka) [inline webpack.config.js]: ah this did it: 'css-loader?localIdentName=[name]__[local]___[hash:base64:5]!autoprefixer-loader'
i think thats what you meant

File: README.md (hunk 2 above)
Comment (hellatan) [inline README.md]: i'm confused b/c it's being required here - https://github.com/1stdibs/dibs-react-tooltip/blob/master/Tooltip.jsx#L8

File: webpack.config.js (hunk 1 above)
Comment (alejandromaka) [inline webpack.config.js]: oh it does work i was using the same logic for the other package which doesnt have the same tools, duh..

File: README.md (hunk 2 above)
Comment (kostasmanionis) [inline README.md]: It's a separate file, it gets swapped by a module that exports an object with the class names. 

`main.css` is a file that's created at pretty much the very end of a webpack build. It's just another asset that webpack exports. The user needs to consume this file on it's own.

**Comment by alejandromaka** [comment]
fixed

File: webpack.config.js (hunk 1 above)
Comment (hellatan) [inline webpack.config.js]: > It's fancy with it's arguments checks
@kostasmanionis do you mean dibs-webpack-scss is? Does it matter if the loaders potentially diverge then?

Comment (kostasmanionis) [inline webpack.config.js]: I was talking about ExtractTextPlugin, missed your comment about `dibs-webpack-scss`. Though keep in mind that `dibs-webpack-scss` is a private package, but the tooltip is public 😄 Doesn't really matter if they diverge.

Comment (hellatan) [inline webpack.config.js]: > dibs-webpack-scss is a private package

ya, aware of that - we'd obviously need to make it public. I guess it doesn't really matter if it does diverge - one less dep to worry about 😄

**Comment by alejandromaka** [comment]
bump :)

**Comment by robrichard** [comment]
do we normally gitignore the dist folder?

**Comment by hellatan** [comment]
> do we normally gitignore the dist folder?

@robrichard - yes as long as there is also a `.npmignore` file so it doesn't get removed from the package.

**Comment by hellatan** [comment]
@alejandromaka i think you're going to have to delete the `dist` folder so it doesn't stay in this repo

**Comment by alejandromaka** [comment]
i think it should be good.. can you check again?

**Pull request merged**